docs/pages/deploy/send-over-the-air-updates.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Send over-the-air updates
+description: Learn how to send over-the-air updates to push critical bug fixes and improvements to your users.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+
+You can send over-the-air updates containing critical bug fixes and improvements to your users.
```

Comment:
```suggestion
You can send over-the-air (OTA) updates containing critical bug fixes and improvements to your users.
```

Should we add the abbreviate form here?

Reply:
I did not since there are no more instances below this of "over-the-air", so there's nowhere to use it. So, I think no unless we add more instances of "over-the-air" later on in the doc. Then I think OTA would be okay to add.